satellite/analytics/service.go

```diff
@@ -114,3 +117,46 @@ func (service *Service) TrackCreateUser(fields TrackCreateUserFields) {
 		Properties: props,
 	})
 }
+
+// TrackSignedIn sends an "Signed In" event to Segment.
```

Comment:
Just a sidenote for future improvements. These events could be specified via structs instead of arguments, two variations:

```
s.analytics.Track(analytics.SignedIn{
	UserID: auth.User.ID,
	Email: auth.User.Email,
})

analytics.SignedIn{
	UserID: auth.User.ID,
	Email: auth.User.Email,
}.Enqueue(s.analytics)
```

This could make the different events more self contained.

CC: @mobyvb

Reply:
Nice, I like this suggestion. I can make a refactor ticket to do this